src/EditorFeatures/CSharpTest/Completion/CompletionProviders/EnumAndCompletionListTagCompletionProviderTests.cs

```diff
@@ -491,5 +491,32 @@ enum E
 ";
             await VerifyNoItemsExistAsync(markup);
         }
+
+        [WorkItem(18359, "https://github.com/dotnet/roslyn/issues/18359")]
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task NotAfterDotWithTextTyped()
+        {
+            var markup =
+@"namespace ConsoleApplication253
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            M(E.a$$)
```

Comment:
probably good to have a comment saying that the normal symbolcompletionprovider gives us the values here, so we don't need to duplicate it.  Also, we probably want a test in symbolcompletionprovidertests to make sure that we *do* get values from it.